src/stories/LineChart.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import { LineChart } from '../chart/LineChart';
+import { Line } from '../cartesian/Line';
+import { data } from './data';
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+// eslint-disable-next-line import/no-default-export
```

Comment:
Can we configure the ESLint rules that don't apply to storybook on a certain path in the eslint config?

Reply:
We may need a `.eslintignore` and add the path for the stories folder
```
# Ignore all files in stories folder
src/stories/*
```

Reply:
For disabled/overrides certain rules for a path, follow:
https://eslint.org/docs/latest/use/configure/configuration-files#how-do-overrides-work

In `.eslintrc.json`:
```json
{
  "rules": {
    "quotes": ["error", "double"]
  },

  "overrides": [
    {
      "files": ["bin/*.js", "lib/*.js"],
      "excludedFiles": "*.test.js",
      "rules": {
        "quotes": ["error", "single"]
      }
    }
  ]
}
```

Reply:
I think eslint should also be effective in stories - after all they are code as well and should conform to best practice.